style.css

```diff
@@ -87,3 +88,212 @@ body {
   align-items: baseline;
   justify-content: flex-start;
 }
+
+.projects-section {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  margin-top: 650px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.cards {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.cards-details {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  width: 295px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.title {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  width: 236px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.highlight-text {
+  width: 55px;
+  height: 16px;
+  text-transform: uppercase;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  flex: none;
+  flex-grow: 0;
+}
+
+.countertext {
+  width: 88px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.cards-descriptions {
+  width: 295px;
+  height: 76px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.grid-skills {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 10px;
+  padding: 10px;
+  align-items: center;
+  list-style: none;
+  margin: -20px 0 0 -10px;
+  gap: 8px;
+}
+
+.grid-skills > div {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  padding: 4px 12px;
+  width: 45px;
+  height: 24px;
+  font-weight: 500;
+  letter-spacing: 0.03rem;
+  line-height: 16px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+}
```

Comment:
Looks nice but I feel like the padding is overused, consider improving on that 😅 take a look at the differences below 👇🏽 

## Figma
![Screenshot 2023-02-17 at 21 35 40](https://user-images.githubusercontent.com/45739963/219827336-6ff85a59-08ac-43e1-9f9e-7eb762eae0af.png)



## Your design
![Screenshot 2023-02-17 at 21 34 24](https://user-images.githubusercontent.com/45739963/219827249-e5d075f8-a92d-4430-84f1-805cd31641fe.png)

Would look nicer, right? 😎